app/client/ui/GristTooltips.ts

```diff
@@ -250,9 +250,9 @@ Only .tar attachment archives downloaded from Grist can be uploaded here."
   formFraming: (...args: DomElementArg[]) => cssTooltipContent(
     cssMarkdownSpan(
       t(
-"This form is created by a Grist user, and is not endorsed by Grist Labs. \
-Do not submit passwords through this form, and be careful with links in \
-it. Report malicious forms to [{{mail}}](mailto:{{mail}}).", {
+"This form is created by a Grist user, and is not endorsed by Grist Labs, Inc. \
+or any party providing this service. For your security, do not submit passwords through this form, \
```

Comment:
Note to the reviewers: I have just obtained a feedback of my team, suggesting that it would be better to explain the purpose of the advise (hence "For your security").